### DISCUSSION TITLE
Missing dependencies on OpenSuse TW

### DISCUSSION THREAD

**Discussion by ybgp** [ciderapp/Apple-Music-Electron#397] [Q&A] [answered]
Hey everyone

I downloaded the .rpm file yesterday and installed it without any problems even though it asked for libuuid and nss packages. I'm using Opensuse TW. Anyone knows where I can get these packages? All I could find was libuuid1 and libuuid1-32bit on the official repos. 
It still works but I'm just wondering if other TW users had the same error message

**Comment by quacksire** [comment, marked as answer]
Try [compiling our source](https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Compiling-on-Platforms) to see if it fixes itself.

**Reply by ybgp** [reply]
Thank you. It worked :)
